fiaas_deploy_daemon/deployer/kubernetes/deployment/deployer.py

```diff
@@ -236,3 +234,12 @@ def _build_fiaas_env(config):
             "CONSTRETTO_TAGS": ",".join(("kubernetes-{}".format(config.environment), "kubernetes", config.environment)),
         })
     return env
+
+
+def _build_global_env(global_env):
+    """
+    global_env key/value are added as is and with the key prefix FIAAS_
```

Comment:
This part was IMHO a misguided attempt at something I don't even remember anymore. Especially the bit about having both with and without the prefix. Either it should only be with `FIAAS_` prefix, or don't mess with them at all.
And again, since it's been like this a while, it might need a deprecation period and proper handling of the migration, so that users are not caught out.

Reply:
Yes, I think it makes sense to remove this behavior; it is mostly just confusing. If the cluster operator wants environment variables prefixed with `FIAAS_` they can just use that as the variable key when adding the variable. Like below I think I'd prefer handing that in a followup PR to this though